testing/spectest/shared/fulu/sanity/block_processing.go

```diff
@@ -35,6 +36,11 @@ func RunBlockProcessingTest(t *testing.T, config, folderPath string) {
 	testFolders, testsFolderPath := utils.TestFolders(t, config, "fulu", folderPath)
 	for _, folder := range testFolders {
 		t.Run(folder.Name(), func(t *testing.T) {
+			params.SetupTestConfigCleanup(t)
+			cfg := params.BeaconConfig().Copy()
+			cfg.BlobSchedule = []params.BlobScheduleEntry{{MaxBlobsPerBlock: 9}}
```

Comment:
the follows electra: https://github.com/OffchainLabs/prysm/blob/99cd90f335e3821f83bc022f6bbcdead616cf452/testing/spectest/shared/electra/sanity/block_processing.go#L41
another way is to set electra fork epoch to 0